includes/functions.php

```diff
@@ -1191,6 +1191,37 @@ function pmpro_changeMembershipLevel( $level, $user_id = null, $old_level_status
 			$pmpro_error = sprintf( __( 'Error interacting with database: %s', 'paid-memberships-pro' ), ( ! empty( $wpdb->last_error ) ? $wpdb->last_error : 'unavailable' ) );
 			return false;
 		}
+
+		/**
+		 * Allow filtering whether to remove duplicate "active" memberships by setting them to "changed".
+		 *
+		 * @since TBD
+		 *
+		 * @param bool $remove_duplicate_memberships Whether to remove duplicate "active" memberships by setting them to "changed".
+		 */
+		$remove_duplicate_memberships = apply_filters( 'pmpro_remove_duplicate_membership_entries', false );
```

Comment:
Let's default to true.
```suggestion
		$remove_duplicate_memberships = apply_filters( 'pmpro_remove_duplicate_membership_entries', true );
```
```suggestion
		$remove_duplicate_memberships = apply_filters( 'pmpro_remove_duplicate_membership_entries', false );
```